waveform.go

```diff
@@ -118,7 +119,7 @@ func plotAudios(imgfile string, audios [][]int16, audionames []string, samplerat
 	for i, audioname := range audionames {
 		plots = append(plots, fmt.Sprintf(
 			`"%s" every 30 using 1:%d title "%s" with lines`,
-			f.Name(),
+			filepath.ToSlash(f.Name()),
```

Comment:
gnuplot dislikes windows path separator (`\`), then requires this conversor that is a no-op on linux.

Reply:
yay windows :D